tests/test_config.py

```diff
@@ -25,6 +23,7 @@ def test_set_base_url():
     new_url = 'another_url'
     config.set_base_url(new_url)
     assert config.get_base_url() == new_url
+    config.set_base_url(None)
```

Comment:
I am not a big fan of resetting stuff after running the test like this. Isn't there anything like setup/ tearDown similar to unittests?